docs/iris/src/whatsnew/2.0a0.rst

```diff
@@ -97,6 +97,12 @@ Incompatible Changes
 
 * The :meth:`~iris.cube.Cube.lazy_data` method no longer accepts any arguments.
 
+* Using :meth:`~iris.cube.Cube.convert_units` on a cube with unknown units will
```

Comment:
I thought about that, but then you wouldn't have the links to the documentation for the methods.  I have no strong opinion either way.